cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -249,7 +249,9 @@ message SearchJobStatusRequest {
         }
     }
 
-    Filter filter = 1;
+    // Used to remove jobs from the returned stream that do not satisfy *all* of the given constraints.
+    // Note that only one of each filter variant may be applied. For example, attempting to jobs that passed *or* failed will be invalid.
```

Comment:
"... attempting to jobs that passed ..."

Reply:
Woops